### ISSUE TITLE
Fee reimbursement and unblock deposit and trade amount for trade 13721289

### ISSUE THREAD

**Issue by fupm** [bisq-network/support#782]
ID: 13721289-d2a148e9-e897-4723-acfc-369f1ffe40bd-151

SCREENSHOT HERE
![13721289](https://user-images.githubusercontent.com/76731476/105469581-c6801800-5c98-11eb-80ab-53f6accbec4c.png)

Maker: fc295afe3c5c2906dd15e0459f0cfeb5b1652c90171c0fb893311c4c0b5caaeb
Taker: 468856d04a1c8e9068abe984f85cdb53d6f954f74a3d16fb2c168cd155211189
Deposit: 3ebc89e8887e3207aea5beea909eb8f17498c329b039f29c1b5df734734b225a

Bisq version: 1.5.2

Other notes:

We never get the confirmation of the blockchain so that the trade never could get started and my BTC deposit and the amount for trading is blocked since then.

Thank you in advance!

**Comment by huey735** [comment]
Please edit the title to match the template: `Fee reimbursement for trade 13721289`
And change the screenshot to the rest of the info of the trade like https://github.com/bisq-network/support/issues/790. But do blank the sensitive information.

**Comment by leo816** [comment]
please provide your preferred btc address for reimbursement

**Comment by fupm** [comment]
bc1qjyq52pnh06yg0n5gzn82s0876d3pptldk8gqgt

**Comment by fupm** [comment]
hello i'm still waiting @leo816 and for another issue https://github.com/bisq-network/support/issues/877
thank you in advanced!

**Comment by huey735** [comment]
@fupm Once again I ask you to edit the title to match the template: `Fee reimbursement for trade 13721289`. Regarding the `locked` part I ask that you refer to this guide https://github.com/bisq-network/support/blob/3c9e55a5b34842dabf12a7fad2a265f9a4ca96bb/docs/known-bugs.md#failedTrade

Feel free to DM me on [Keybase}(https://keybase.io/team/bisq). I'm huey735.

**Comment by pazza83** [comment]
Closing as no response
